docs/technical-documentation/rest-patch.md

```diff
@@ -0,0 +1,96 @@
+# Updating resources - PATCH request
+
+PATCH requests allow you to update resources in place via a REST call. In this case you send a few required fields and then any changed fields. PATCH requests are available for nodes and media entities, but _not_ for files. PATCH requests are very consistent between node and media entities so we will just show one set of examples here.
+
+Our example node is at `http://localhost:8000/node/3`.
+
+If you perform a [GET](./rest-get.md) against another node or media you can see all of the fields, some of these are calculated (change based on others, ie. "changed"), some are references to other entities (nodes,media,files in Drupal. ie. "field_model") and others are pure text fields (ie. "title"). 
+
```

Comment:
Add spaces after commas, e.g. `nodes,media,files` should be `nodes, media, files`.

Reply:
done